journals/apps/journals/models.py

```diff
@@ -445,8 +470,24 @@ class JournalIndexPage(Page):
         index.SearchField('search_description', partial_match=True)
     ]
 
+    api_fields = [
+        APIField('intro'),
+        APIField('hero_image_url')
+    ]
+
+    @property
+    def hero_image_url(self):
+        if self.hero_image:
+            return self.hero_image.file.url  # pylint: disable=no-member
+        else:
+            return ''
+
+    def get_journal(self):
```

Comment:
I realize you're just using this to get an example journal so you can pull the site from it, but I don't like the idea of an Index page having a `get_journal` method because it's not intuitive what it would do.